gradle.properties

```diff
@@ -7,6 +7,7 @@ cordaProductVersion = 5.0.0
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
+cordaApiRevision = 631
 cordaApiRevision = 630
```

Comment:
Did this render wrong or am I seeing an extra line instead of a replaced one?

Reply:
you're correct, merge error, will update